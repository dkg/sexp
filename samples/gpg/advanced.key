(protected-private-key (ecc (curve Curve25519)(flags djb-tweak)(q
  #400A5A713520D9CBC9A4F46897A4906430FFDD3AB8AD4012FD57E0196284E56F31#)
 (protected openpgp-s2k3-ocb-aes ((sha1 #78481D15B3D9817B#
  "5991424")#F0C778361CED4B54FBD76F84#)#E6DB31D5439985FD89B8E607055E1B5
CA3457EA56545A1E149517E703B27FA78195B5A9DA6A088B65E86E1681F7BB5F94690A
7EFBEAF7737D27847F9#)(protected-at "20221110T201113")))