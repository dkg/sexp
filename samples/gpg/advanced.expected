(protected-private-key
 (ecc
  (curve Curve25519)
  (flags djb-tweak)
  (q |QApacTUg2cvJpPRol6SQZDD/3Tq4rUAS/VfgGWKE5W8x|)
  (protected
   openpgp-s2k3-ocb-aes
   ((sha1 |eEgdFbPZgXs=| "5991424") |8Md4NhztS1T712+E|)
   |5tsx1UOZhf2JuOYHBV4bXKNFfqVlRaHhSVF+cDsn+ngZW1qdpqCItl6G4Wgfe7X5RpCn776vdzfSeEf5|)
  (protected-at "20221110T201113")))
